vaadin-list-mixin.html

```diff
@@ -208,18 +210,27 @@
       return -1;
     }
 
+    _setHidden(item) {
+      if (getComputedStyle(item).getPropertyValue('display') === 'none') {
+        item.hidden = true;
+      }
+    }
+
     _setFocusable(idx) {
-      idx = this._getAvailableIndex(idx, 1, item => !item.disabled);
+      idx = this._getAvailableIndex(idx, 1, item => !(item.disabled || getComputedStyle(item).getPropertyValue('display') === 'none'));
```

Comment:
Instead of repeating `getComputedStyle(item).getPropertyValue('display') === 'none'` in multiple places, it's probably better to create a `_isHidden(item)` method that checks it instead

Reply:
done